service/src/main/java/org/jboss/aerogear/unifiedpush/service/impl/spring/OAuth2Configuration.java

```diff
@@ -23,6 +23,9 @@
 	private static final String KEY_OAUTH2_ADMIN_USERNAME = "aerogear.config.oauth2.admin.username";
 	private static final String KEY_OAUTH2_ADMIN_PASSWORD = "aerogear.config.oauth2.admin.password";
 
+	private static final String KEY_OAUTH2_MASTER_ADMIN_USERNAME = "aerogear.config.oauth2.master.admin.username";
```

Comment:
why `KEY_OAUTH2_MASTER_ADMIN_USERNAME` and not `KEY_OAUTH2_MASTER_RELAM_ADMIN_USERNAME`  or `KEY_MASTER_RELAM_ADMIN_USERNAME` as requested in the issue?

same question for `aerogear.config.oauth2.master.admin.username` vs `aerogear.config.master.realm.admin.username`